### PR TITLE
tools: update the usage of region startkey to region keys

### PR DESCRIPTION
**Pull request by IcePigZDB**
Signed-off-by: IcePigZDB <icepigzdb@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Support endKey in http interface `regions/key` and adjust the pdctl region command `region startKey` to `region keys`accordingly. This is originally proposed as a precondition of the push down optimisation of `infomation_schema.tikv_region_status`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs/pull/6951
- Other reference link(s):
    * tracking issue: tikv/pd/issues/4400

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@IcePigZDB Since you have added a tickbox v5.4, is this a v5.4 feature?

**Comment by IcePigZDB** [comment]
> @IcePigZDB Since you have added a tickbox v5.4, is this a v5.4 feature?

yes, if anything change, I will keep you informed.

**Review by nolouch — APPROVED** [review]

**Comment by nolouch** [comment]
PTAL @TomShawn

File: pd-control.md [hunk 1]
```diff
@@ -806,16 +806,46 @@ Encoding 格式示例：
 }
 ```
 
-### `region startkey [--format=raw|encode|hex] <key> <limit>`
+### `region keys [--format=raw|encode|hex] <start_key> <end_key> <limit>`
 
-用于查询从某个 key 开始的所有 Region。
+用于查询从某个 key 范围内的所有 Region。支持不带 endKey 的范围, limit 的默认值是16，不带 endKey 的默认值是-1(没有限制)。示例如下。
```
Comment (TomShawn) [inline pd-control.md:811]:
```suggestion
用于查询从某个 key 范围内的所有 Region。支持不带 `endKey` 的范围。`limit` 的默认值是 16，不带 `endKey` 时的默认值是 `-1`（表示没有限制)。示例如下：
```

File: pd-control.md [hunk 2]
```diff
@@ -806,16 +806,46 @@ Encoding 格式示例：
...
 
-示例：
+显示从 a 开始的所有 Region 信息，数量上限为16：
```
Comment (TomShawn) [inline pd-control.md:813]:
```suggestion
显示从 a 开始的所有 Region 信息，数量上限为 16：
```

File: pd-control.md [hunk 3]
```diff
@@ -806,16 +806,46 @@ Encoding 格式示例：
...
+```
+
+```
+{
+  "count": 16,
+  "regions": [......],
+}
+```
+
+显示从 a 开始的所有 Region 信息，数量上限为20：
```
Comment (TomShawn) [inline pd-control.md:828]:
```suggestion
显示从 a 开始的所有 Region 信息，数量上限为 20：
```

File: pd-control.md [hunk 4]
```diff
@@ -806,16 +806,46 @@ Encoding 格式示例：
...
+```
+
+```
+{
+  "count": 20,
+  "regions": [......],
+}
+```
+
+显示[a, z) 范围内的所有 Region 信息，没有数量上限：
```
Comment (TomShawn) [inline pd-control.md:843]:
```suggestion
显示 [a, z) 范围内的所有 Region 信息，没有数量上限：
```

File: pd-control.md [hunk 5]
```diff
@@ -825,6 +855,21 @@ Encoding 格式示例：
 }
 ```
 
+显示[a, z) 范围内的所有 Region 信息，数量上限为20：
```
Comment (TomShawn) [inline pd-control.md:858]:
```suggestion
显示 [a, z) 范围内的所有 Region 信息，数量上限为 20：
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**